objects/official/scenery_small/toontowner.scenery_small.xxbbbr01_fix/object.json

```diff
@@ -9,7 +9,7 @@
     "objectType": "scenery_small",
     "properties": {
         "price": 20,
-        "removalPrice": 0,
+        "removalPrice": 15,
```

Comment:
You do not seem to have tested this (or compared with the other base blocks). I suggest you do, it will make the error quite apparent ;-)

Reply:
I don’t know how to test the object. How do I download it from my fork? I don’t know how to put the json with the day since on my Mac it has to split the parkobj to be able to access the json

Reply:
In that case, just take a look at one of the other base blocks, specifically their `removalPrice`.

Reply:
Thank you for your patience and understanding. This is a big learning curve for me with no coding/github experience. I think I got it fixed now.

Reply:
No worries, I very deliberately challenged you.